app/bundles/EmailBundle/Entity/EmailRepository.php

```diff
@@ -536,6 +536,8 @@ public function resetVariants($relatedIds, $date)
     /**
      * Up the read/sent counts.
      *
+     * @depreacated use upCountSent or incrementRead method
```

Comment:
There is a typo here in the word deprecated

Reply:
Thanks, fixed

Reply:
I don't even know how I ended up in this repo, but you're welcome 😅🫱🏻‍🫲🏼

Reply:
Haha you're always welcome @exsesx 🚀 Thanks!